config/dispatcher/dispatcher.ex

```diff
@@ -504,6 +504,14 @@ defmodule Dispatcher do
     Proxy.forward conn, [], "http://digital-signing/signing-flows/" <> signing_flow_id <> "/pieces/" <> piece_id <> "/signinghub-url"
   end
 
+  post "/signing-flows/reset-signflow/:signing_flow_id", @json_service do
+    Proxy.forward conn, [], "http://digital-signing/signing-flows" <> signing_flow_id
+  end
+
```

Comment:
see frontend PR.  this rule should not be needed

Reply:
was going to do this change and then I see that this dispatcher call could have never worked.
It should have forwarded to `"http://digital-signing/signing-flows/reset-signflow" <> signing_flow_id` to match the endpoint in the service

Removing anyway